components/eamxx/src/share/field/field_layout.cpp

```diff
@@ -43,8 +46,17 @@ int FieldLayout::get_vector_dim () const {
   return std::distance(m_tags.cbegin(),it);
 }
 
+// get the extent of the CMP (Components) tag in the FieldLayout
+// e.g., for FieldLayout f({COL, CMP, LEV}, {ncol, ncmp, nlev});
+// we have get_component_idx(f) = ncmp
```

Comment:
I think this comment got messed up during copy+paste. you probably meant `we have get_vector_dim() = ncmp`?

Reply:
oof, twice in a row 🤦‍♂️

fixed!